src/SystemViewToolbar.tsx

```diff
@@ -0,0 +1,209 @@
+// import LockIcon from "@mdi/svg/svg/lock-outline.svg";
+import {Lock as LockIcon} from "@emotion-icons/bootstrap/Lock";
+// import MinusIcon from "@mdi/svg/svg/minus.svg";
+import {Minus as MinusIcon} from "@emotion-icons/boxicons-regular/Minus";
+// import PlusIcon from "@mdi/svg/svg/plus.svg";
+import {Plus as PlusIcon} from "@emotion-icons/boxicons-regular/Plus";
+// import SelectionIcon from "@mdi/svg/svg/checkbox-multiple-marked-outline.svg";
+import {SelectMultiple as SelectionIcon} from "@emotion-icons/boxicons-regular/SelectMultiple";
+// import UnlockIcon from "@mdi/svg/svg/lock-open-variant-outline.svg";
+import {Unlock as UnlockIcon} from "@emotion-icons/bootstrap/Unlock";
+
+export type Props = {
+  nodes: any[]
+  edges: any[]
+  lrOrientation: boolean
+  fitViewParams?: FitViewParams
+  onZoomIn?: () => void
+  onZoomOut?: () => void
+  onFitview?: () => void
+  onInteractiveChange?: (isInteractive: boolean) => void
+  onToggleOrientation?: (lrOrientation: boolean) => void
+};
+
+
+export const SystemViewToolbar: React.FC<Props> = (props: Props) => {
+
+  const [lrOrientation, setLROrientation] = useState<boolean>(props.lrOrientation);
+
+  let defaultSelectedTab: string | undefined;
+  const [selectedTab, setSelectedTab] = React.useState(defaultSelectedTab);
+
+  let defaultSelectedTab2: string | undefined;
+  const [selectedTab2, setSelectedTab2] = React.useState(defaultSelectedTab2);
+
+  const GroupingOptions = {
+    first: { id: "logical", label: "Logical", },
+    second: { id: "physical", label: "Physical", },
+    third: { id: "none", label: "None", },
+  };
+
+  const [selectedId, setSelectedId] = React.useState(GroupingOptions.third.id);
+  const optionArray: Option[] = Object.values(GroupingOptions);
+
+  const { zoomIn, zoomOut, fitView } = useZoomPanHelper();
+  const setInteractive = useStoreActions((actions) => actions.setInteractive);
+  const isInteractive = useStoreState((s) => s.nodesDraggable && s.nodesConnectable && s.elementsSelectable);
+
+  const onToggleOrientation = useCallback(() => {
+    setLROrientation(!lrOrientation);
+    props.onToggleOrientation?.(lrOrientation);
+  }, [lrOrientation, props.onToggleOrientation]);
+
+  const onZoomInHandler = useCallback(() => {
+    zoomIn?.();
+    props.onZoomIn?.();
+  }, [zoomIn, props.onZoomIn]);
+
+  const onZoomOutHandler = useCallback(() => {
+    zoomOut?.();
+    props.onZoomOut?.();
+  }, [zoomOut, props.onZoomOut]);
+
+  const onFitViewHandler = useCallback(() => {
+    fitView?.(props.fitViewParams);
+    props.onFitview?.();
+  }, [fitView, props.fitViewParams, props.onFitview]);
+
+  const onInteractiveChangeHandler = useCallback(() => {
+    setInteractive?.(!isInteractive);
+    props.onInteractiveChange?.(!isInteractive);
+  }, [isInteractive, setInteractive, props.onInteractiveChange]);
+
+  return (
+    <Toolbar>
+      <br />
+      <ExpandingToolbar
+        tooltip="Group nodes"
+        iconName={
```

Comment:
I had to replace `icon` with `iconName`. It doesn't seem `icon` is available as a type here. Maybe I'm missing something.